Jenkinsfile

```diff
@@ -44,6 +44,36 @@ pipeline {
                     }
                 }
 
+                stage('ubuntu-18.04/gcc-7.3.0 (Debug/Coverage/unittest)') {
+                    agent {
+                        docker {
+                            image 'ubuntu:bionic'
+                            args '--cap-add sys_ptrace'
+                        }
+                    }
+                    steps {
+                        sh 'echo $NODE_NAME'
+                        sh 'echo y | sudo ./script/installation/packages.sh'
+                        sh 'sudo apt-get -y install curl lcov'
+                        sh 'mkdir build'
+                        sh 'cd build && cmake -DCMAKE_BUILD_TYPE=Debug -DTERRIER_USE_ASAN=OFF -DTERRIER_GENERATE_COVERAGE=ON -DTERRIER_BUILD_BENCHMARKS=OFF .. && make -j24'
+                        sh 'cd build && timeout 1h make unittest'
+                        sh 'cd build && timeout 1h make check-tpl'
+                        sh 'cd build && python ../script/testing/junit/run_junit.py'
+                        sh 'cd build && lcov --directory . --capture --output-file coverage.info'
+                        sh 'cd build && lcov --remove coverage.info \'/usr/*\' --output-file coverage.info'
+                        sh 'cd build && lcov --remove coverage.info \'*/build/*\' --output-file coverage.info'
+                        sh 'cd build && lcov --remove coverage.info \'*/third_party/*\' --output-file coverage.info'
+                        sh 'cd build && lcov --remove coverage.info \'*/benchmark/*\' --output-file coverage.info'
+                        sh 'cd build && lcov --remove coverage.info \'*/test/*\' --output-file coverage.info'
+                        sh 'cd build && lcov --remove coverage.info \'*/src/main/*\' --output-file coverage.info'
+                        sh 'cd build && lcov --list coverage.info'
+                        sh 'cd build && curl -s https://codecov.io/bash > ./codecov.sh'
+                        sh 'cd build && chmod a+x ./codecov.sh'
+                        sh 'cd build && /bin/bash ./codecov.sh -X gcov'
```

Comment:
Try changing this to:
`sh 'cd build && /bin/bash ./codecov.sh -X gcov -U "--proxy http://proxy.pdl.cmu.edu:3128/"'`

Reply:
Before you push any changes, I've got a test build (http://jenkins.db.cs.cmu.edu:8080/blue/organizations/jenkins/terrier/detail/PR-551/18/pipeline) in progress now that the PR is reopened.  If that works, then you could push the change I suggested.